homeassistant/components/garmin_connect/config_flow.py

```diff
@@ -37,11 +38,15 @@ async def async_step_user(self, user_input=None):
         if user_input is None:
             return await self._show_setup_form()
 
-        garmin_client = Garmin(user_input[CONF_USERNAME], user_input[CONF_PASSWORD])
+        websession = async_get_clientsession(self.hass)
+
+        garmin_client = Garmin(
+            websession, user_input[CONF_USERNAME], user_input[CONF_PASSWORD]
+        )
 
         errors = {}
         try:
-            await self.hass.async_add_executor_job(garmin_client.login)
+            unique_id = await garmin_client.login()
```

Comment:
Is this the same unique id as before?

Reply:
If this is the username I'd name the variable to that so it's easier to know what kind of unique id we're using.

Reply:
Yes more clear, renamed